.gitignore

```diff
@@ -8,9 +8,9 @@
 *_cache/
 /cache/
 *_files/
-template-computo-R.html
+ArticleGeomFPOP.html
 logo_text_white.pdf
-template-computo-R.pdf
+ArticleGeomFPOP.pdf
```

Comment:
why file name  "ArticleGeomFPOP" and not "computo_pishchagina_changepoints"?

Reply:
It is a mistake, sorry about that.